cmd/root/sweet.go

```diff
@@ -36,6 +36,20 @@ var Cmd = &cobra.Command{
 	},
 }
 
+// Exercises that should be added to the
+// default exercises directory if it's not present
+// in
+var defaultExercises = []Exercise{
+	{
+		name: "hey.go",
+		text: "fmt.Println(\"hey\")\n",
+	},
+	{
+		name: "hey.js",
+		text: "console.log('hey')\n",
```

Comment:
Change these to an actual interesting exercise.

I thought about having the default exercises be files in a directory in this project, but I'm not sure if the code would break if I tried loading files instead of just strings. Perhaps I can try that out later.